### ISSUE TITLE
Failed to run (wine) Python pip pefile...

### ISSUE THREAD

**Issue by Seek4samurai** [Veil-Framework/Veil#475]
Hi, I'm using Ubuntu 22.04 version and I cloned the repository to install Veil. 
To install Veil I followed
```
git clone https://github.com/Veil-Framework/Veil.git
cd Veil/
./config/setup.sh --force --silent
```
## Error
```
 [*] Finished Veil configuration...



 [*] Finished environment checks

 [ERROR] There was issues installing the following:

 
Failed with apt-get update (1): 0
Failed to run (wine) Python pip pefile... Exit code: 1



 [I] If you have any errors running Veil, run: './Veil.py --setup' and select the nuke the wine folder option



 [I] Done!

(base) seek4samurai@ubuntu:~/Downloads/Veil$ veil
veil: command not found
```
After proceeding with installation it gives following errors at the end.

**Comment by xkill** [comment]
On Kali:
```
 [*] Installing (Wine) Python's PIP pefile

Collecting pip==19.1.*
  Downloading https://files.pythonhosted.org/packages/5c/e0/be401c003291b56efc55aeba6a80ab790d3d4cece2778288d65323009420/pip
-19.1.1-py2.py3-none-any.whl (1.4MB)
    100% |################################| 1.4MB 351kB/s 
Installing collected packages: pip
  Found existing installation: pip 7.1.2
    Uninstalling pip-7.1.2:
      Successfully uninstalled pip-7.1.2
Successfully installed pip-19.1.1
You are using pip version 19.1.1, however version 22.2.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.                                        
DEPRECATION: Python 3.4 support has been deprecated. pip 19.1 will be the last one supporting it. Please upgrade your Python
 as Python 3.4 won't be maintained after March 2019 (cf PEP 429).
Collecting future
  Downloading https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/fut
ure-0.18.2.tar.gz (829kB)
     |████████████████████████████████| 829kB 6.8MB/s
Installing collected packages: future
  Running setup.py install for future ... \0158:err:winediag:ntlm_check_version ntlm_auth was not found or is outdated. Make sure that ntlm_auth >= 3.0.25 is in your path. Usually, you can find it in the winbind package of your distribution.
0158:err:ntlm:ntlm_LsaApInitializePackage no NTLM support, expect problems
done
Successfully installed future-0.18.2
DEPRECATION: Python 3.4 support has been deprecated. pip 19.1 will be the last one supporting it. Please upgrade your Python
 as Python 3.4 won't be maintained after March 2019 (cf PEP 429).
Collecting pefile
  Downloading https://files.pythonhosted.org/packages/f9/1e/fc4fac0169d16a98577809400bbcfac8ad1900fa792184327b360ea51fc6/pef
ile-2021.5.13.tar.gz (66kB)
     |████████████████████████████████| 71kB 5.1MB/s
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):                                                                               
      File "<string>", line 1, in <module>                                                                                  
      File "C:\users\root\Temp\pip-install-gkynggai\pefile\setup.py", line 86, in <module>                                  
        long_description = "\n".join(_read_doc().split('\n')),                                                              
      File "C:\users\root\Temp\pip-install-gkynggai\pefile\setup.py", line 33, in _read_doc                                 
        tree = ast.parse(f.read())                                                                                          
      File "C:\Python34\lib\ast.py", line 35, in parse                                                                      
        return compile(source, filename, mode, PyCF_ONLY_AST)                                                               
      File "<unknown>", line 3789                                                                                           
        f'Export directory contains more than 10 repeated entries '                                                         
                                                                  ^                                                         
    SyntaxError: invalid syntax                                                                                             
    ----------------------------------------                                                                                
ERROR: Command "python setup.py egg_info" failed with error code 1 in C:\users\root\Temp\pip-install-gkynggai\pefile\       
 [ERROR] Failed to run (wine) Python pip pefile... Exit code: 1  
 ```

**Comment by xkill** [comment]
Worked if you fix the version of pefile to `2019.4.18`: line 587 to:
```bash
  sudo -u "${trueuser}" WINEPREFIX="${winedir}" wine "${winedir}/drive_c/Python34/python.exe" "-m" "pip" "install" "pefile==2019.4.18"
```

**Comment by Seek4samurai** [comment]
> Worked if you fix the version of pefile to `2019.4.18`: line 587 to:
> 
> ```shell
>   sudo -u "${trueuser}" WINEPREFIX="${winedir}" wine "${winedir}/drive_c/Python34/python.exe" "-m" "pip" "install" "pefile==2019.4.18"
> ```

I had to manually install Wine64 architecture and then it worked :)
and then going into the Veil directory and running it manually.
